intel/k8s.py

```diff
@@ -213,6 +224,13 @@ def get_namespaces(config):
     return k8s_api.list_namespace().to_dict()
 
 
+# Get kubelet version from node
+def get_kubelet_version(config):
+    k8s_api = version_api_client_from_config(config)
+    version_info = k8s_api.get_code()
+    return int(version_info.major), int(version_info.minor)
```

Comment:
This raises an error on my setup as my minor version is "7+".
Error: Traceback (most recent call last):                                                                                         
  File "/cmk/cmk.py", line 145, in <module>                                                                                
    main()                                                                                                                 
  File "/cmk/cmk.py", line 122, in main                                                                                    
    args["--publish"])                                                                                                     
  File "/cmk/intel/reconcile.py", line 49, in reconcile                                                                    
    version_major, version_minor = k8s.get_kubelet_version(None)                                                           
  File "/cmk/intel/k8s.py", line 231, in get_kubelet_version                                                               
    return int(version_info.major), int(version_info.minor)                                                                
ValueError: invalid literal for int() with base 10: '7+'